clc/paths/paths.go

```diff
@@ -0,0 +1,87 @@
+package paths
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	unixDir       = ".local/share/clc"
+	EnvCLCHome    = "CLC_HOME"
+	DefaultConfig = "config.yaml"
+)
+
+func Home() string {
+	dir := os.Getenv(EnvCLCHome)
+	if dir == "" {
+		cd, err := os.UserHomeDir()
+		if err != nil {
+			panic(err)
+		}
+		dir = filepath.Join(cd, unixDir)
```

Comment:
As far as I remember, we did research about directories on different OS and in here we are using the UNIX style. Will it be changed later for Windows?

Reply:
Added that at: https://github.com/hazelcast/hazelcast-commandline-client/pull/160/commits/b9fd102d81c6100a8d5beef5b705fbf6f42f464e

Reply:
Added that at: https://github.com/hazelcast/hazelcast-commandline-client/pull/160/commits/b9fd102d81c6100a8d5beef5b705fbf6f42f464e